package.json

```diff
@@ -28,39 +28,37 @@
     "npm": ">=9"
   },
   "dependencies": {
-    "@types/mapbox-gl": "^2.7.11",
-    "geostyler-style": "^7.3.1"
+    "@types/mapbox-gl": "^2.7.18",
+    "geostyler-style": "^7.5.0"
   },
   "scripts": {
     "build-browser": "webpack --config browser-build.config.js",
     "build-dist": "tsc -p tsconfig.json",
-    "build:browser": "webpack --config browser-build.config.js",
     "build": "npm run build-dist && npm run build-browser",
     "lint:test:build": "npm run lint && npm run test && npm run build",
     "lint:test": "npm run lint && npm run test",
     "lint": "eslint -c .eslintrc.js --ext .ts . && tsc --noEmit --project tsconfig.json",
     "prepublishOnly": "npm run build",
-    "release": "np --no-yarn && git push https://github.com/geostyler/geostyler-mapbox-parser.git master --tags",
-    "start:dev": "webpack --config dev-build.config.js --watch",
+    "release": "np --no-yarn && git push upstream master --tags",
```

Comment:
Not a fan, this only works if the remote named `upstream` points at the right location. Have you thought about this?

Reply:
Yes. The previous version does not work at all anymore. And https://github.com/geostyler/geostyler-mapbox-parser/pull/298 basically replaces the whole workflow